tests/controls/dataviewctrltest.cpp

```diff
@@ -102,24 +102,32 @@ class DataViewCtrlTestModel: public wxDataViewModel
     //     |-- wxTEST_ITEM_CHILD
     //         |
     //         |-- wxTEST_ITEM_GRANDCHILD
+    //         |   |
+    //         |   |-- wxTEST_ITEM_LEAF
+    //         |   |
+    //         |   |-- wxTEST_ITEM_FIRST_HIDDEN
     //         |
-    //         |-- wxTEST_ITEM_GRANDCHILD_HIDDEN
+    //         |-- wxTEST_ITEM_SECOND_HIDDEN
```

Comment:
I think remaining this made diff much larger than necessary. Could you please leave the old name for this one and rename `wxTEST_ITEM_FIRST_HIDDEN` to `wxTEST_ITEM_LEAF_HIDDEN` to keep the names consistent?